src/helpers/figures/charts/chart_ui_common.ts

```diff
@@ -176,6 +177,12 @@ export function getChartLabelValues(
   labelRange?: Range
 ): LabelValues {
   let labels: LabelValues = { values: [], formattedValues: [] };
+  if (dataSets.length === 1) {
+    const data = getData(getters, dataSets[0]);
+    if (data.length && data.every((e) => typeof e === "string" && !isEvaluationError(e))) {
+      return { values: data, formattedValues: data };
+    }
+  }
```

Comment:
I don't think this is needed anymore. At least no test fail if I remove this locally.